test.js

```diff
@@ -13,3 +13,12 @@ test(t => {
 	t.is(m('🐶', 5), '🐶');
 	t.is(m('👍🏿', 1), '👍🏻');
 });
+
+test('Color name constants', t => {
```

Comment:
`xo` complained, so I've added this 😄 
How would you name the other one, I don't have any ideas 😅